apps/web/styles/globals.css

```diff
@@ -44,8 +44,6 @@
   --cal-brand-text: white;
 }
 .dark {
-  color-scheme: dark;
```

Comment:
@sean-brydon I removed this in the PR. Though it should go live as a separate hotfix once you confirm